src/pages/scientificServices/pipelines/views/RunJob.tsx

```diff
@@ -227,7 +228,7 @@ export const RunJob = () => {
                 })}
 
               {/* Displays optional run description */}
-              <PipelineRunDescription value={runDescription} onChange={setRunDescription} />
+              {selectedPipeline && <PipelineRunDescription value={runDescription} onChange={setRunDescription} />}
```

Comment:
drive-by fix: description text box shouldn't be visible until a pipeline is selected